data/json/items/armor.json

```diff
@@ -503,7 +503,7 @@
         "volume" : 8,
         "warmth" : 30,
         "environmental_protection" : 2,
-        "encumbrance" : 10,
+        "encumbrance" : 18,
```

Comment:
"Durable and comfortable" in the description. They are pretty thin and have low armor values. Compare them to regular boots.